src/main/java/gift/domain/cart/CartItemController.java

```diff
@@ -3,67 +3,57 @@
 import gift.domain.product.Product;
 import gift.domain.user.dto.UserInfo;
 import gift.global.resolver.LoginInfo;
-
-import gift.global.response.ResponseMaker;
-import gift.global.response.ResultResponseDto;
-import gift.global.response.SimpleResultResponseDto;
-import java.util.List;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.jdbc.core.JdbcTemplate;
-import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestParam;
 
-@RestController
-@RequestMapping("/api/users")
+@Controller
+@RequestMapping("/users/cart")
 public class CartItemController {
 
     private final CartItemService cartItemService;
-    private final JdbcTemplate jdbcTemplate;
 
-    public CartItemController(CartItemService cartItemService, JdbcTemplate jdbcTemplate) {
+    public CartItemController(CartItemService cartItemService) {
         this.cartItemService = cartItemService;
-        this.jdbcTemplate = jdbcTemplate;
-    }
-
-
-    /**
-     * 장바구니에 상품 담기
-     */
-    @PostMapping("/cart/{id}")
-    public ResponseEntity<SimpleResultResponseDto> addCartItem(
-        @PathVariable("id") Long productId, @LoginInfo UserInfo userInfo) {
-
-        cartItemService.addCartItem(userInfo.getId(), productId);
-
-        return ResponseMaker.createSimpleResponse(HttpStatus.OK, "상품이 장바구니에 추가되었습니다.");
     }
 
-    /**
-     * 장바구니 조회
-     */
-    @GetMapping("/cart")
-    public ResponseEntity<ResultResponseDto<List<Product>>> getProductsInCartByUserId(
+    @GetMapping
+    public String cartPage(
+        Model model,
+        @RequestParam(value = "page", defaultValue = "0") int page,
+        @RequestParam(value = "sort", defaultValue = "id_asc") String sort,
         @LoginInfo UserInfo userInfo) {
```

Comment:
취향 문제일 수 있어서 따로 코멘트는 안남겼는데, 이미 수정하셨으니 적어보면
저는 실무에서 이러한 방식으로 개행합니다ㅎㅎ

```suggestion
        @LoginInfo UserInfo userInfo
    ) {
```

Reply:
수정하니 더욱 깔끔한 것 같습니다. 공유 감사합니다!